src/api/clearlyDefined.js

```diff
@@ -16,6 +16,14 @@ export const ORIGINS_NUGET = 'origins/nuget'
 export const ORIGINS_MAVEN = 'origins/maven'
 export const ORIGINS_PYPI = 'origins/pypi'
 export const ORIGINS_RUBYGEMS = 'origins/rubygems'
+export const ORIGINS = {
```

Comment:
This feels leaky. Users of the clearlydefined api should just indicate what they want and not have to pass in elements of the url to be accessed. So in `getRevisions` it would be better for users to pass in what they have and we figure out the right URL to use in the request.

Reply:
Right, i've updated the method avoiding the user to pass in directly URL informations.